### PR TITLE
changed status response code to 401 from 403 in the illustration

### PR DESCRIPTION
**Pull request by Biki-das**
> What was wrong/why is this fix needed? (quick summary only)
the illustration showed a wrong response code of 403 , it should be a response of code 401

> Anything else that could help us review it

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (teoli2003) [inline .vscode/settings.json:1]: This file shouldn't be in the PR.

**Review by teoli2003 — CHANGES_REQUESTED** [review]
More generally, I believe that 403 is a perfectly valid answer from the server.

https://httpwg.org/specs/rfc7231.html#status.403

> The 403 (Forbidden) status code indicates that the server understood the request but refuses to authorize it. A server that wishes to make public why the request has been forbidden can describe that reason in the response payload (if any).

> If authentication credentials were provided in the request, the server considers them insufficient to grant access. The client SHOULD NOT automatically repeat the request with the same credentials. The client MAY repeat the request with new or different credentials. However, a request might be forbidden for reasons unrelated to the credentials.

> An origin server that wishes to "hide" the current existence of a forbidden target resource MAY instead respond with a status code of 404 (Not Found)

More information there: https://www.freecodecamp.org/news/http-401-error-vs-http-403-error-status-code-responses-explained/

I'm not against changing the image, but I think we should make it clear that 401, 403 and even 404 are perfectly valid answers here.

**Comment by teoli2003** [comment]
Also the font in the image is not correct; you should either use Times New Roman or copy the 401 that is used above.

**Comment by teoli2003** [comment]
(And this PR would fix #7240 )

**Comment by Biki-das** [comment]
Ok thanks, i will surely do thag

**Comment by Biki-das** [comment]
Thank you teoli, i am a beginner to open source and i am grateful for this kind of guidance and i will be making the changes as you said

**Comment by Biki-das** [comment]
hello teoli, as you said we should make it clear that 401,403 and 404 are valid responses, should i add a new paragraph for that or where should i add these piece of information and make these required changes

**Comment by teoli2003** [comment]
You are very welcome: take your time to make the changes, don't be afraid of making errors.

Yes, feel free to add a new paragraph and/or modify the existing ones explaining how 401, 403, 404 can be returned, and when.

**Comment by Biki-das** [comment]
i have corrected the illustration image, please check it

**Comment by teoli2003** [comment]
The file is fine. But as the tests are indicating, you'll need to reupload it once you "compressed" it. 

That is, you need to run the command line: `yarn filecheck "files/en-us/web/http/authentication/httpauth.png" --save-compression` then upload the commit.

Also, we can add/update the paragraph about 401/403/404 here, or in another PR, like you want (I would prefer here, but your choice here)

**Comment by Biki-das** [comment]
Good afternoon  teoli, looks like i am messing up to much , could you guide me ,the workflow before commiting like, what things to commit and what not to, the package.json seems to getting in the commit how do i avoid that?

**Comment by teoli2003** [comment]
H! There should only be 2 files in your PR: the image, and the index.html of the page.

So you need to remove `yarn.lock`from the PR; to do that, you should `git reset`it to the version in `upstream main`:

`git checkout main yarn.lock`in `mdn/content`, then push a commit with this change only and this should be better.

Don't forget to check the preview URL linked above by the github bot, it often allows you to find a lot of typos difficult to find by reading the html.

**Comment by Biki-das** [comment]
Thanks a lot @teoli2003 , i am utterly grateful, for your support and feedback you are giving me, open source is fun and interesting

**Comment by Biki-das** [comment]
> Thanks a lot @teoli2003 , i am utterly grateful, for your support and feedback you are giving me, open source is fun and interesting, if you find any thing for a beginner like me, ping me up anytime or if you want any help, i am available!

**Comment by Biki-das** [comment]
hello @teoli2003  i have made the changes and fixed the system status error , please review my last commit

**Comment by Biki-das** [comment]
hello @teoli2003 , please give me a feedback

File: files/en-us/web/http/authentication/index.html [hunk 2]
```diff
@@ -34,7 +34,7 @@ <h3 id="Proxy_authentication">Proxy authentication</h3>
 
 <h3 id="Access_forbidden">Access forbidden</h3>
 
-<p>If a (proxy) server receives valid credentials that are inadequate to access a given resource, the server should respond with the {{HTTPStatus("403")}} <code>Forbidden</code> status code. Unlike {{HTTPStatus("401")}} <code>Unauthorized</code> or {{HTTPStatus("407")}} <code>Proxy Authentication Required</code>, authentication is impossible for this user.</p>
+<p>If a (proxy) server receives valid credentials that are inadequate to access a given resource, the server should respond with the {{HTTPStatus("403")}} <code>Forbidden</code> status code. Unlike {{HTTPStatus("401")}} <code>Unauthorized</code> or {{HTTPStatus("407")}} <code>Proxy Authentication Required</code>, authentication is impossible for this user.the 401 response points that the request lacks the required authentication credentials, here the user may send a new request or replace the existing Authorization header field</p>
```
Comment (teoli2003) [inline files/en-us/web/http/authentication/index.html:37]: What about:

```suggestion
<p>If a (proxy) server receives <em>invalid</em> credentials, it should return a {{HTTPStatus("401")}} <code>Unauthorized</code> or a {{HTTPStatus("407")}} <code>Proxy Authentication Required</code>, and the user may send a new request or replace the {{HTTPHeader("Authorization")}} header field.</p>
<p>If a (proxy) server receives valid credentials that are <em>inadequate</em> to access a given resource, the server should respond with the {{HTTPStatus("403")}} <code>Forbidden</code> status code. Unlike {{HTTPStatus("401")}} <code>Unauthorized</code> or {{HTTPStatus("407")}} <code>Proxy Authentication Required</code>, authentication is impossible for this user and browsers will not propose a new attempt.</p>
<p>In all cases, the server may prefer returning a {{HTTPStatus("404")}} <code>Not Found</code> status code, to hide the existence of the page to user without adequate privileges or not correctly authenticated.</p>
```
I prefer this order (401 is more common than 403, and 404 needs to be mentioned).

**Review by teoli2003 — CHANGES_REQUESTED** [review]
The image looks good. I tried to improve the text to make it even clearer.

File: files/en-us/web/http/authentication/index.html (hunk 2 above)
Comment (Biki-das) [reply files/en-us/web/http/authentication/index.html:37]: Thanks a lot, teoli, i will surely change the above as you said

**Comment by Biki-das** [comment]
Hello @teoli2003 , i have taken your commit suggestions and applied them ,again thanks for guiding me up , i hope my PR gets merged this time

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me: it was a really difficult task for a first PR! Congratulations! 

Welcome aboard!

**Comment by Biki-das** [comment]
thanks a lot, i am very happy and this is possible becuase of your guidance @teoli2003 , this is just the beginning i will be contrbuting more and as much as i can,.

**Comment by Biki-das** [comment]
hello @teoli2003 , i am trying to run the mdn content repo in my local machine to do more stuff but i am getting this error,  


💔 Loading error - Google Chrome 09-08-2021 21_50_05

**Comment by teoli2003** [comment]
I'm not a specialist of Yari, but have you checked if the directory where you cloned mdn/content has a whitespace (even escapted) in the path?

**Pull request merged**